src/win/process.c

```diff
@@ -1110,18 +1143,18 @@ int uv_spawn(uv_loop_t* loop,
     }
   }
 
-  if (process_flags & CREATE_SUSPENDED) {
-    if (ResumeThread(info.hThread) == ((DWORD)-1)) {
+  if (process_flags & CREATE_SUSPENDED && hThread != NULL) {
```

Comment:
```suggestion
  if ((process_flags & CREATE_SUSPENDED) && hThread != NULL) {
```
Having said that.... UV_PROCESS_DETACHED and UV_PROCESS_WINDOWS_RUNAS_ADMIN should probably be mutually exclusive if they don't work together?

Reply:
how? is that means if two specified together, we return an error OR we ignore one of them, if the former one, which error should be returned?

Reply:
Specifying them both should be an error (and should be documented as such), probably UV_EINVAL.

Reply:
done